client/src/components/ProductDetail/ProductDetail.js

```diff
@@ -65,7 +68,14 @@ const ProductContainer = styled.section`
   display: grid;
   gap: 10px;
   margin-bottom: 20px;
+
+  @media (min-width: 992px) {
+    grid-template-columns: 1fr 1fr;
+  }
 `
+
+const ProductContent = styled.div``
+
 const BackButton = styled.div`
```

Comment:
Why did you use a div with role="button" here? You could style it exactly the same way using a real <button> element, since a div should only be used, if no other element fits - or simple change it to `const BackButton = styled(NavLink)` and remove `as={NavLink}` from the code above.

Reply:
that's a good solution